tools/STOC/STOCepsen.xml

```diff
@@ -0,0 +1,131 @@
+    ]]></help>
+
+    <citations>
+        <citation type="bibtex">
+            @article{article,
+            author = {Fontanilles, Philippe and Van Acker, Bernard},
+            year = {2016},
+            pages = {68-86},
+            title = {Avifaune nicheuse d’une jeune forêt de plaine du Pays Basque. Paramètres démographiques dans le cadre d’une évolution rapide du milieu},
+            volume = {16},
+            booktitle = {Casseur d'os},
+            url = {https://crbpo.mnhn.fr/IMG/pdf/fontanilles_2016_casseur_d_os_stoc_briscous.pdf}
+            }
+        </citation>
+        <citation type="doi">10.1111/j.2041-210X.2012.00207.x</citation>  
+        <citation type="doi">10.2193/2006-328</citation> 
+        <citation type="bibtex">
+            @article{article,
+            author = {Fontanilles, Philippe and Grabières, Ganix and Fourcade, Jean-Marc and Van Acker, Bernard and Urbina-Tobias, Patrice},
+            year = {2015},
+            month = {12},
+            pages = {73-99},
+            title = {Avifaune nicheuse d'une lande atlantique de la côte basque, structure démographique et tendance temporel le.},
+            volume = {15},
+            booktitle = {Casseur d'os},
+            url = {https://crbpo.mnhn.fr/IMG/pdf/fontanilles_2015_casseur_d_os_bilan_stoc_abbadia.pdf}
+            }
+        </citation>
+        <citation type="bibtex">
+            @article{article,
+            author = {Delorme, Quentin},
+            year = {2013},
+            month = {01},
+            pages = {11-19},
+            title = {DIX-SEPT ANNÉES DE SUIVI TEMPOREL DES OISEAUX COMMUNS PAR BAGUAGE (STOC CAPTURE) À LIART (ARDENNES). QUELLES ÉVOLUTIONS ET TENDANCES DE LA POPULATION ?},
+            booktitle = {In'Fox Bulletin annuel du Regroupement des Naturalistes ARDenais},
+            url = {https://crbpo.mnhn.fr/IMG/pdf/Delorme_2014_INFOX_bilan_STOC.pdf}
+            }
+        </citation>
+        <citation type="bibtex">
+            @techreport{techreport,
+            author = {Jiguet, Frédéric},
+            year = {2005},
+            title = {Bilan du programme STOC pour la France en 2004},
+            url = {https://crbpo.mnhn.fr/IMG/pdf/Jiguet_2005_ORNITHOS_bilan_STOC.pdf}
+            }
+        </citation>
+        <citation type="bibtex">
+            @techreport{techreport,
+            author = {Julliard, Romain},
+            year = {2001},
+            title = {Oiseaux communs en France  : Variations d’abondance entre 1989 et 1998. Evaluation du programme STOC-capture},
+            url = {https://crbpo.mnhn.fr/IMG/pdf/julliard_2001_alauda_bilan_stoc.pdf}
+            }
+        </citation>
+        <citation type="bibtex">
```

Comment:
This can be reused by macros https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-macros in many of your tools